HerculesStudio-manifest.yaml

```
@@ -0,0 +1,21 @@
+---
+app-id: com.jacobdekel.HerculesStudio
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: HerculesStudio
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
```

```suggestion
- "--socket=fallback-x11"
```
assuming this supports Wayland.